database/database_test.go

```diff
@@ -201,41 +201,41 @@ func TestGetRequestsToJoinTeams(t *testing.T) {
 	t.Run("get requests returns de-duplicated requests", func(t *testing.T) {
 		database := New(testhelpers.Maketemp(t))
 
-		addRequestToJoinToDatabase(t, newestReq, database)
-		addRequestToJoinToDatabase(t, oldestReq, database)
-		addRequestToJoinToDatabase(t, anotherReq, database)
+		addRequestToJoinToDatabase(t, req1, database)
+		addRequestToJoinToDatabase(t, newerReq1, database)
+		addRequestToJoinToDatabase(t, req2, database)
 
 		gotRequests, err := database.GetRequestsToJoinTeams()
 		assert.NoError(t, err)
 
-		expectedRequests := []team.RequestToJoinTeam{oldestReq, anotherReq}
+		expectedRequests := []team.RequestToJoinTeam{req2, newerReq1}
 		assert.Equal(t, len(expectedRequests), len(gotRequests))
 		assert.Equal(t, expectedRequests, gotRequests)
 	})
 
-	t.Run("returns oldest when newest was inserted first", func(t *testing.T) {
+	t.Run("returns newest when oldest was inserted first", func(t *testing.T) {
 		database := New(testhelpers.Maketemp(t))
-		addRequestToJoinToDatabase(t, newestReq, database)
-		addRequestToJoinToDatabase(t, oldestReq, database)
+		addRequestToJoinToDatabase(t, req1, database)
+		addRequestToJoinToDatabase(t, newerReq1, database)
 
 		gotRequests, err := database.GetRequestsToJoinTeams()
 		assert.NoError(t, err)
 
-		expectedRequests := []team.RequestToJoinTeam{oldestReq}
+		expectedRequests := []team.RequestToJoinTeam{newerReq1}
 		assert.Equal(t, len(expectedRequests), len(gotRequests))
 		assert.Equal(t, expectedRequests, gotRequests)
 	})
 
 	t.Run("returns oldest when oldest was inserted first", func(t *testing.T) {
```

Comment:
I think this test should read "returns *newest* when *newest* was inserted first"

Reply:
I've nested the two cases to make it clearer